docker-compose-prod.yml

```diff
@@ -3,8 +3,8 @@ services:
   app:
     build: .
     environment:
-      - S3_EXTENSIONS_BUCKET_HOST=brave-core-ext.s3.brave.com
-      - S3_EXTENSIONS_BUCKET_HOST_TOR=tor.bravesoftware.com
+      - S3_EXTENSIONS_BUCKET_HOST=brave-ext.s3.brave.com
+      - S3_EXTENSIONS_BUCKET_HOST_TOR=brave-ext-tor.brave.com
```

Comment:
tor still has to be on `bravesoftware.com` because `brave.com` was blocked by OpenDNS

Reply:
ooh, can we use the s3 url like in dev - brave-ext-tor.s3-us-west-2.amazonaws.com?

Reply:
that's not cached so we still need our Fastly service in front